api/database/factories/PoolFactory.php

```diff
@@ -43,9 +43,10 @@ public function definition()
 
     public function configure()
     {
+        // take(5) will grab the first 5 classifications, in this case, all the IT classifications, restricting search to IT classifications
         return $this->afterCreating(function (Pool $pool) {
             $assets = CmoAsset::inRandomOrder()->limit(4)->get();
-            $classifications = Classification::inRandomOrder()->limit(3)->get();
+            $classifications = Classification::take(5)->get();
```

Comment:
This is a bit dangerous.  You're depending on the classifications to always be seeded a certain way and the database behaving in a certain way.  The assumptions could very easily break.